### DISCUSSION TITLE
not download. why?

### DISCUSSION THREAD

**Discussion by kyhyponponpain** [aclap-dev/video-downloadhelper#541] [Help and Q&A]
### Your question

https://b9dm.org/
Downloading from this site fails.

### Addon & CoApp version and details

{
  "version": "9.0.1.13",
  "target": "google",
  "channel": "stable",
  "page_url": "https://b9dm.org/20211.html",
  "error": {
    "details": "SideDownload error: : [out#0/mp4 @ 0000022f1ae36700] Output file does not contain any stream\r\nError opening output file K:\\anime\\ 第7話 B9 Dailymtion Youtube B9dmアニメ.mp4.\r\nError opening output files: Invalid argument\r\n",
    "downloadable_id": "downloadable_8370635153674927",
    "error": "coapp_failure",
    "id": "coapp_failure_35422671-5c67-4197-b439-7bdf5b318b6b"
  }
}

**Comment by GrampaWildWilly** [comment]
I do not work for VDH.  I'm just another user.

I visited that page & also got an error trying to download the content with VDH.  So I looked at the Network Monitor.  A file with the extension .m3u8 did show up.  But there were a couple of problems with it.  First, it was identified with Type html.  This is incorrect.  HLS master manifests are supposed to by of Type x-mpegurl.  I went ahead & tried to run ffprobe on the file anyway.  The web site gave me a 403 Forbidden error.  That means the web site has protected this critical file.  My conclusion is that the web site has very effectively thwarted downloading without using DRM.  In short, VDH is not going to work here.  I suspect no other tool is going to work either.  But you are certainly welcome to research that for yourself.
